server/src/services/consentService.ts

```diff
@@ -0,0 +1,27 @@
+import { Consent } from '../models/consent';
+import { getRepository, In } from 'typeorm';
+
+export async function captureConsent(consent: Consent) {
```

Comment:
recently I decided to start work with DB normally and clean the mess that we have  right now.  Idea is to   adopt using repository pattern for DB access. Example here: `rsschool-app/server/src/repositories/interview.ts`

Would be great if you convert this service to `repository`